frontend/my-app/src/GenMeetingData.js

```diff
@@ -0,0 +1,22 @@
+export const GenMeetingData = [
+      {
```

Comment:
Organization wise, I would create one file and place all the dummy data there, rather than have two different files of dummy data

Reply:
My bad- I used those files for testing and forgot to delete them. Will go in and remove those files tmrw! The app gets its data from the backend now.